.travis.yml

```diff
@@ -17,61 +17,89 @@ jobs:
     - env: TOXENV=docs
 
     - stage: tests
-      env: TOXENV=py27-t310-c45
+      env: TOXENV=py27-t310-c45-xdist22
```

Comment:
hmm probably a tad too many envs

Reply:
no actually it seems like travis copes with them fine

Reply:
..but it's really not necessary.
And it blocks all other jobs from the pytest-dev org.

Reply:
Ok I've changed the config to only add one additional test run